hw/ip/rv_core_ibex/doc/boot-rom-patching.md

```diff
@@ -175,51 +175,67 @@ In this case, the dedicated SRAM can be locked after being loaded so that the pa
 
 A patch is a short sequence of instructions that replace some from the original code.
 The replacement is performed by re-routing instruction fetches to fetch from alternative addresses.
-This re-routing is handled by a block that sits between the Ibex core and TL-UL crossbar.
-The block is configured by 32 *patch match registers*.
+This re-routing is handled by a logic block that sits between the Ibex core and TL-UL crossbar.
+The block is configured by 32 sets of 4 match registers:
+- [`IBUS_ADDR_MATCHING`](../data/rv_core_ibex.hjson#ibus_addr_matching_0)
+- [`IBUS_ADDR_EN`](../data/rv_core_ibex.hjson#ibus_addr_en_0)
+- [`IBUS_REMAP_ADDR`](../data/rv_core_ibex.hjson#ibus_remap_addr_0).
+- [`IBUS_REGWEN`](../data/rv_core_ibex.hjson#ibus_regwen_0).
 
-A patch match register is structured as follows.
-The patch applies to a small region of code (4, 8, 16 or 32 bytes), starting at a given address.
-This region is encoded in a field called `M_BASE`.
-The destination of the redirection is given by its address, encoded in a field called `R_BASE`.
+The patch loader is responsible for configuring those registers for each patch in the patch table.
+Each entry in the OTP ROM patch table is composed of two 32 bit words with the following layout:
 
-- If the source region has address `yyy...yyy`, the `M_BASE` field is:
+![Patch match register layout](patch-match.svg)
+
+- The first 32 bits describe the patch match configuration:
```

Comment:
Would it make sense to either define "patch match" somewhere or to tweak the text to avoid using the term?

Reply:
I tweaked the text to `The first 32 bits describe the ROM code region to be patched:` because that's actually what it really is.